pkgs/build-support/setup-hooks/move-docs.sh

```diff
@@ -6,18 +6,17 @@ preFixupHooks+=(_moveToShare)
 
 _moveToShare() {
     forceShare=${forceShare:=man doc info}
-    if [ -z "$forceShare" -o -z "$out" ]; then return; fi
+    if [[ -z "$forceShare" || -z "$out" ]]; then return; fi
```

Comment:
```suggestion
    if [[ -z $forceShare || -z $out ]]; then return; fi
```

Reply:
I would tend to quote whenever possible to offer a consistent way of doing things. A little bit like the use of `const` and `var` in other languages. So that when the quotes are missing, it means the reader should really pay attention.

Reply:
You're right, shellcheck doesn't complain about this one.